src/main/java/uk/gov/ons/census/action/model/dto/FieldworkFollowup.java

```diff
@@ -30,4 +30,5 @@
   private String surveyName;
   private Boolean undeliveredAsAddress;
   private Boolean blankQreReturned;
+  private boolean handDelivery;
```

Comment:
Again, it's all about whether we should make this `Boolean` and nullable. Depends on various idodcy currently ongoing